blocks/gu_spoverview/block_gu_spoverview.php

```diff
@@ -43,13 +52,90 @@ public function hide_header() {
      *       See myoverview plugin for reference.
      */
     public function get_content() {
-        if ($this->content !== null) {
-          return $this->content;
+        global $USER, $DB, $PAGE, $OUTPUT;
+
+
+        $PAGE->requires->css('/blocks/assessmentglance/styles.css');
```

Comment:
Calls a CSS file from a different plugin. Please call the file /blocks/gu_spoverview/styles.css